Recipe/index.html

```diff
@@ -1,57 +1,68 @@
 <!DOCTYPE html>
+
 <html>
+
     <head>
         <title>Recipe for Vegan Cornbread</title>
+        <title>Confetti Background</title>
         <link href="style.css" rel="stylesheet">
-        <style>
-          h3 {
-            color:aqua;
-          }
-        </style>
+        <link href="https://fonts.googleapis.com/css2?family=Bitter&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Great+Vibes&display=swap" rel="stylesheet">
     </head>
+
 <body>
-  <div class="top">
-    <h1 style = 'color:wheat;'>Delicious Vegan Cornbread</h1>
-    <p>There is a vegan restaurant 5 mins away from my house.  This restaurant serves the best vegan cornbread.  It is delicious and moist.  I wanted to find a similar recipe, so I could make it at home and save some money!</p>
+
+  <div class="TOPtitleAndParagraph">
+    <container id="container1">
+    <h1 id="h1Title">Vegan Cornbread Recipe!!</h1>
+    </container>
+    <p id="paragraph">Who doesn't love cornbread?!  I know I do! Espcecially, when it is delicious and moist.  Below is a simple recipe for VEGAN cornbread.  Enjoy!</p>
   </div>
 
-  <div class="left">
+  <div class="LEFTimgPCIngre">
+
     <div>
-        <img src="https://mindfulavocado.com/wp-content/uploads/2021/09/vegan-cornbread_06.jpg" alt="A scrumptious piece of cornbread with butter on top" width="600">
-        <p>Photo credit <a href="https://mindfulavocado.com/vegan-cornbread/" title="Visit Mindful Avocado Website">Mindful Avocado</a></p>
-    </div>
+        <img src="https://mediavine-res.cloudinary.com/image/upload/s--3J8MVuhv--/c_limit,f_auto,fl_lossy,h_1080,q_auto,w_1920/v1555518939/cczyssoxtfdogtw3l3t8.jpg" alt="A scrumptious piece of cornbread with butter on top" width="600">
+        <p id="photoCredit">Photo credit <a href="https://mindfulavocado.com/vegan-cornbread/" title="Visit Mindful Avocado Website">Mindful Avocado</a></p>
+        <!-- https://healthyhappylife.com/best-vegan-cornbread/ -->
+      </div>
     
-    <div>
-      <h3>Ingredients</h3>
-      <ul>
-        <li>1 1/4 cups all purpose flour</li>
-        <li>1 cup yellow corn meal</li>
-        <li>2/3 cup granulated sugar</li>
-        <li>1 teaspoon salt</li>
-        <li>1 tablespoon baking powder</li>
-        <li>1 1/4 cup unsweetened almond milk</li>
-        <li>1/3 cup canola oil</li>
-      </ul>
-    </div>
-  </div>
+    <div class="test">
```

Comment:
Consider changing the class name `test` to something more meaningful and descriptive. This also means changing the `.test` in your style.css file to match the change.